Library/Homebrew/software_spec.rb

```diff
@@ -353,7 +354,7 @@ class BottleSpecification
   def initialize
     @rebuild = 0
     @prefix = Homebrew::DEFAULT_PREFIX
-    @cellar = Homebrew::DEFAULT_CELLAR
+    @cellar = nil # Legacy single cellar for all the tags
```

Comment:
It's not clear to me what this comment means.

Reply:
This has been changed to a clearer variable name.